### PR TITLE
ISSUE-141 Add a static informative page for 401 error

### PR DESCRIPTION
**Pull request by priyank5485**

### CODE REVIEW

**Comment by priyank5485** [comment]
Closes https://github.com/hortonworks/registry/issues/141

File: webservice/src/main/resources/public/assets/401.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, shrink-to-fit=no, initial-scale=1">
+<meta name="description" content="">
+<meta name="author" content="">
+<title>Streamline</title>
```
Comment (shahsank3t) [inline webservice/src/main/resources/public/assets/401.html]: nit: Need to change the title to "Registry"

Comment (priyank5485) [reply webservice/src/main/resources/public/assets/401.html]: Thanks for the catch. Updated.

**Comment by priyank5485** [comment]
@satishd I have rebased this PR. Can you please review and merge this if you find okay? @shahsank3t has already approved it and its tested on a kerberos setup. Thanks

**Comment by priyank5485** [comment]
@satishd I have addressed your review comment.

**Pull request closed without merging**